engine/rpcserver_test.go

```diff
@@ -1688,3 +1688,66 @@ func TestGetManagedOrders(t *testing.T) {
 		t.Errorf("unexpected order result: %v", oo)
 	}
 }
+
+func TestRPCServer_unixTimestamp(t *testing.T) {
+	t.Parallel()
+
+	s := RPCServer{
+		Engine: &Engine{
+			Config: &config.Config{
+				RemoteControl: config.RemoteControlConfig{
+					GRPC: config.GRPCConfig{
+						TimeInNanoSeconds: false,
+					},
+				},
+			},
+		},
+	}
+	const sec = 1618888141
+	const nsec = 2
+	x := time.Unix(sec, nsec)
+
+	timestampSeconds := s.unixTimestamp(x)
+	if timestampSeconds != sec {
+		t.Errorf("have %d, want %d", timestampSeconds, sec)
+	}
+
+	s.Config.RemoteControl.GRPC.TimeInNanoSeconds = true
+	timestampNanos := s.unixTimestamp(x)
+	if want := int64(sec*1_000_000_000 + nsec); timestampNanos != want {
+		t.Errorf("have %d, want %d", timestampSeconds, want)
+	}
+}
+
+func TestRPCServer_GetTicker_LastUpdatedNanos(t *testing.T) {
```

Comment:
If you want to be more precise you can try deploying a dummy ticker:

```go 

+       tn := time.Now()
+       base := currency.NewCode("SILLY")
+       quote := currency.NewCode("NESS")
+       err := ticker.ProcessTicker(&ticker.Price{
+               ExchangeName: testExchange,
+               Pair:         currency.NewPair(base, quote),
+               AssetType:    asset.Spot,
+               LastUpdated:  tn,
+       })
+       if err != nil {
+               t.Fatal(err)
+       }

// then below:

	if tn.Unix() != one.LastUpdated {
		t.Errorf("have %d, want %d", one.LastUpdated, tn.Unix())
	}
	if tn.UnixNano() != two.LastUpdated {
		t.Errorf("have %d, want %d", two.LastUpdated, tn.UnixNano())
	}
```

You just need to add the currency pair to the available and enabled list.